src/components/AchievementBadge/AcheivementBadge.test.jsx

```diff
@@ -1,5 +1,5 @@
 import { act } from "@testing-library/react";
-import AchievementBadge from "./AchievementBadge";
+import {AchievementBadge} from "./AchievementBadge";
```

Comment:
This change looked odd to me. Turns out `AchievementBadge` is being exported both as a named export and as the default export. Would you mind changing it so that only one export method is used? (I tend to use default exports for components that live in their own file). And then update any broken imports to reflect that change.

Reply:
Yeah, will do